cmd/root.go

```diff
@@ -58,14 +59,18 @@ func init() {
 }
 
 func initConfig() {
-	if cfgFile == "" {
-		return
-	}
-	viper.SetConfigFile(cfgFile)
-	viper.SetConfigType("yaml")
-	if err := viper.ReadInConfig(); err != nil {
-		logger.Fatal("error while reading the config: %v", err)
+	if cfgFile != "" {
+		cfgFile, err := util.GetResolvedPath(cfgFile)
```

Comment:
Currently, the helpdoc for cfgFile says that it accepts only absolute path.  Should we change that?